src/tqdm.ts

```diff
@@ -24,7 +27,7 @@ export class _tqdmIterator implements Iterator<any> {
         }
     }
 
-    public next(): IteratorResult<boolean, any> {
+    public next(): IteratorResult<_tqdmIteratorResult<T>, undefined> {
```

Comment:
It's the correct form of IteratorResult: the first argument of the type points to the type that will be returned when we have values (`done: false`), the second – to the value when iterator is done (`done: true`).